packages/documentation/src/pages/getting-started/common-tasks/workflow/write.mdx

```diff
@@ -0,0 +1,39 @@
+# Write
+
+## Choose an issue to work on
+
+- Work with your team to determine the right issue to work on
+
+_The VA handbook has more information about the broader process for delivering services to Veterans on VA.gov. Developers work in an agile framework, where issues are assigned to sprints and developers work on this items over the two-week sprint period._
+
+## Create feature branch
+
+- Checkout `master` branch and pull the latest
+```bash
+git checkout master && git pull origin master
+```
+
+- Create a new feature branch from master
+```bash
+git checkout -b 12345-issue-title
+```
+
+_Since most new work derives from a GitHub Issue, it's recommended to prefix your branch name with the issue. For example, if you're working on a feature to collect a veteran's address as detailed in Issue #25, you might call your issue `25-address`._
```

Comment:
looked through `vets-website` recent branches and almost no one followed the suffix recommendation with initials. I updated this to reflect the more common approach but open to changing this back or to something else.